lesson_7/ilas_lorena/decorators_2.py

```diff
@@ -0,0 +1,24 @@
+# Create a decorator called safe_divide that will output a message if the division cannot be performed, othervise it
+# will return the result.
+
+
+def safe_divide(fnc):
+    def check_result(arg1, arg2):
+
+        try:
+            result = arg1 / arg2
+            return result
```

Comment:
Is this assignment really needed here?

Reply:
It is not, I wanted to use result for something more but in the end i changed my mind and left it like this. I will update my code accordingly.